cmd/sqlflow/model_zoo_client_test.go

```diff
@@ -125,7 +93,7 @@ func caseReleaseModel(t *testing.T) {
 	a.NoError(prepareModel())
 	cmd := fmt.Sprintf(
 		`--model-zoo-server=localhost:%d --data-source=%s release model %s v1.0`,
-		modelZooServerPort, database.GetTestingMySQLURL(), "test_model_db.my_model")
+		modelZooServerPort, database.GetTestingMySQLURL(), "iris.my_model")
```

Comment:
Should also remove the `test_model_db` in e2e test https://github.com/sql-machine-learning/sqlflow/blob/b550e427d002aef1dbe18a90c6886203694744a5/cmd/sqlflowserver/e2e_common_cases.go#L66

Reply:
Done.